index.html

```diff
@@ -195,7 +195,7 @@ <h2 class="headline-heading heading-large">About Myself</h2>
               <img
                 class="about-skill-body-img"
                 src="./assets/Ellipse javascript 1.png"
-                alt="js"
+                alt="javascript"
               />
               <span class="about-skill-text">JavaScript</span>
             </li>
```

Comment:
- [ ] Great work on your project so far! I noticed that you have a navigation section in your code, which is an essential part of any website. However, to improve accessibility for all users, especially those who rely on keyboard navigation, I recommend using anchor tags (`<a>`) instead of list items (`<li>`) for your navigation links.

   By using anchor tags, you allow users to navigate through the links using the tab key, making it easier for keyboard-only users and individuals who use assistive technologies. Here's how you can modify your code:

```html
<ul class="nav-links">
  <li><a href="/portfolio">Portfolio</a></li>
  <li><a href="/about">About</a></li>
  <li><a href="/contact">Contact</a></li>
</ul>
```
instead of this 👇🏾 

<img width="355" alt="image" src="https://github.com/Afsheen96/Portfolio/assets/84156714/746d7ea0-36ff-4404-9ba6-3b8493bc07b0">


   With this change, the navigation links will now be accessible using the tab key, and users will be able to navigate to different sections of your website easily.

Remember to update the href attribute of each anchor tag with the appropriate URLs corresponding to your portfolio, about, and contact sections.

Keep up the good work, and don't hesitate to reach out if you have any questions or need further assistance with accessibility or any other aspect of your project.